src/VectorizeLoops.cpp

```diff
@@ -514,6 +514,10 @@ class VectorSubs : public IRMutator {
     }
 
     Expr visit(const Call *op) override {
+        user_assert(!op->is_intrinsic(Call::debug_to_file))
```

Comment:
It's not that the Func is vectorized, it's that it's compute_at inside some other Func's vectorized loop.